scripts/StkAaveScript.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
```

Comment:
If this contract is made for deploying the new impl, it should be a script, and have broadcast and so on. if not you will not be able to execute it